public/electron.js

```diff
@@ -0,0 +1,49 @@
+const electron = require('electron');
+const app = electron.app;
+const session = electron.session;
+const BrowserWindow = electron.BrowserWindow;
+
+const path = require('path');
+const isDev = require('electron-is-dev');
+
+let mainWindow;
+
+function createWindow() {
+  mainWindow = new BrowserWindow({
+    width: 900,
+    height: 680,
+    webPreferences: {
+      // webSecurity: false,
+      // devTools: isDev
+    }
+  });
+  mainWindow.loadURL(isDev ? 'http://localhost:3000' : `file://${path.join(__dirname, '../build/index.html')}`);
```

Comment:
```suggestion
  mainWindow.loadURL(isDev ? 'http://localhost:3000' : `file://${path.join(__dirname, '..', 'build','index.html')}`);
```
path.join же нужен, чтобы от слэшей не зависеть?

Reply:
path.join нужен чтобы склеить dirname c путем до index.html, что так что так
на выходе будет одинаково
я сделал как в примере указано

Reply:
Он также решает проблему разных ОС. В винде и маке/линуксе слэши в разные стороны, поэтому рекомендуется склеивать, а не слэшами.